tests/common/mac/ProjectTestHelpers.cs

```diff
@@ -174,7 +175,18 @@ public static string GenerateEXEProject (UnifiedTestConfig config)
 			WriteMainFile (config.TestDecl, config.TestCode, true, config.FSharp, Path.Combine (config.TmpDir, config.FSharp ? "Main.fs" : "Main.cs"));
 
 			string sourceDir = FindSourceDirectory ();
-			File.Copy (Path.Combine (sourceDir, "Info-Unified.plist"), Path.Combine (config.TmpDir, "Info.plist"), true);
+			var projectPlistPath = Path.Combine(config.TmpDir, "Info.plist");
```

Comment:
Please refactor this chunk of code into a method like CopyPList or something. I'm sure we'll need to reuse it at some point for another test.